docs/src/13_annex.adoc

```diff
@@ -0,0 +1,80 @@
+ifndef::imagesdir[:imagesdir: ../images]
+
+[[section-annex]]
+== Annex
+=== Functional Requirements
+==== Users Sign up.
+[none or no-bullet]
+* FR-USU 1. The system will offer to an unregistered user a registration form.
```

Comment:
Very design dependant, just write "The system shall allow an unregistered user to register in the application", the means of providing the information is irrelevant

Reply:
I agree. Does this mean unregistered must always be logged in, even in the chance they want to just global statistics? I don't remember addressing this point previously.

Reply:
What I meant is: have we discussed this or is this a requirement?